Tests/Recurly/Coupon_Test.php

```diff
@@ -90,10 +90,20 @@ public function testPlanCodesXml() {
     $coupon = Recurly_Coupon::get('special', $this->client);
 
     $this->assertInstanceOf('Recurly_Coupon', $coupon);
-    $this->assertTrue($coupon->applies_to_all_plans);
```

Comment:
Why is this test being changed?

Reply:
According to the docs, `plan_codes` is an "Array of plan_codes the coupon applies to, if applies_to_all_plans is false." So here, `applies_to_all_plans` should actually be false.